### ISSUE TITLE
Using text from one icinga object, colour from another

### ISSUE THREAD

**Issue by s2156945** [meerkat-dashboard/meerkat#148]
**Describe the bug**
The V3 behaviour is different when it comes to service filters. V2 uses the "filter" value for the colour and the "id" for the text.
This may have been an accidental feature but it is used fairly heavily.

**Expected behavior**
It should be possible to replicate the V2 behaviour in V3.

**Screenshots**

Here the V2 output is looking at the "Check SBS OTT Service Health Live" check as per the filter.
Below the V3 output seems to be looking at the objectName "Check SBS OTT Service Health VOD" which is in the objectName element which would have been copied from the "id" element in the V2 config, left populated even though not selected in the V2 edit GUI.

![image](https://user-images.githubusercontent.com/21114973/222353005-9a6a58da-f083-43ba-80ad-24c2c1ae1911.png)

This is the V3 config derived from the V2 config
```
woody@willie:~/work/git/meerkat/cmd/meerkat$ jq .elements[43] < dashboards/sbs-ott-live.json 
{
  "options": {
    "backgroundColor": "#007bff",
    "checkDataDefault": "Overview/Health",
    "checkDataPattern": "/none to match here/",
    "checkDataSelection": "pluginOutput",
    "dynamicText": "",
    "dynamicText2": "",
    "dynamicText2Structure": false,
    "filter": "service.__name==\"TBSP-PELDVIcingaMon02!Check SBS OTT Service Health Live\"",
    "fontColor": "#ffffff",
    "fontSize": 20,
    "linkURL": "https://tbs-dashboards.reportal.broadcastservices.cloud/view/sbs-overview",
    "muteAlerts": false,
    "nameFontSize": 40,
    "objectName": "TBSP-PELDVIcingaMon02!Check SBS OTT Service Health VOD",
    "objectType": "service",
    "selection": "services",
    "textAlign": "center",
    "textVerticalAlign": "center"
  },
  "rect": {
    "h": 6.38977635782748,
    "w": 16.1579892280072,
    "x": 17.5044883303411,
    "y": 1.75718849840256
  },
  "title": "Overview/Health link",
  "type": "check-card"
}

```

**Comment by ollytom** [comment]
This use case I think would be covered by the feature implemented in #132.
One approach would be to set a custom variable to the desired value and read from that.
Objects have the attributes "name" and "display_name".
name is the unique identifier and display_name is generally used by Icingaweb for human readability in its GUI.
So a var named "short_display_name" seems clear in purpose without tying it to Meerkat.
For example:

	object Service "example" {
		dsiplay_name = "Example Overview Service Health"
		check_command = "dummy"
		...
		vars.short_display_name = "Overview/Health"
	}

In Meerkat, instead of selecting the object's state for the card's text, we would select "short_display_name".

**Comment by s2156945** [comment]
We have used this in weird ways not covered by your workaround, sorry.
